Source/Protocols/Plugins/FeaturePlugin/FocusablePlugin/FocusableStrategy/Strategy/CompositFocusableTableStrategy.swift

```diff
@@ -0,0 +1,36 @@
+//
+//  CompositFocusableTableStrategy.swift
+//  ReactiveDataDisplayManager
+//
+//  Created by porohov on 01.12.2021.
+//
+
+import UIKit
+
+public final class CompositFocusableStrategy<CollectionType>: FocusableStrategy<CollectionType> {
```

Comment:
```suggestion
public final class CompositeFocusableStrategy<CollectionType>: FocusableStrategy<CollectionType> {
```
Composit - композит
Composite - композитный


Прилагательным правильнее будет)

Reply:
Документации не хватает.
Напиши, что в этой стратегии можно сочетать несколько стратегий